mods/ts/maps/sunstroke/rules.yaml

```diff
@@ -0,0 +1,36 @@
+World:
+	GlobalLightingPaletteEffect:
+		Red: 1.6
+		Blue: 0.9
+		Green: 1.2
+		Ambient: 0.3
+	LuaScript:
+		Scripts: sunstroke.lua
+
+GAOLDCC2:
+	Capturable:
+	ProvidesPrerequisite@BuildingName:
+
+GAOLDCC3:
```

Comment:
That would make them use the grey neutral player colour on all maps... do we want that?